fcl/caf/cafmakerjob_icarus.fcl

```diff
@@ -47,5 +47,9 @@ physics:
   end_paths:     [ stream1 ]
 }
 
+# MCT0Offset for MC ()
+# ref1 : https://github.com/SBNSoftware/icaruscode/blob/v09_37_02_01/icaruscode/CRT/crtsimmodules_icarus.fcl#L11
+# ref2 : https://github.com/SBNSoftware/icaruscode/blob/v09_64_01/icaruscode/CRT/crttruehitproducer.fcl#L8
+physics.producers.cafmaker.CRTSimT0Offset: 1600000
```

Comment:
Proposed alternative:
* `#include "crtsimmodules_icarus.fcl"`
* ```suggestion
  physics.producers.cafmaker.CRTSimT0Offset: @local::standard_icarus_crtsimalg.GlobalT0Offset
   ```

Reply:
Thanks Gianluca, your suggestion is way better! I updated it as you recommended, and checked correct number is put.